app/src/main/java/com/infomaniak/mail/ui/newMessage/NewMessageFragment.kt

```diff
@@ -272,14 +273,20 @@ class NewMessageFragment : Fragment() {
             shouldDisplayCloseButton = true,
             onDelete = ::onDeleteAttachment,
             onAttachmentClicked = {
-                trackAttachmentActionsEvent(OPEN_FROM_DRAFT_NAME)
-                it.openAttachment(
-                    context = requireContext(),
-                    navigateToDownloadProgressDialog = { attachment, attachmentIntentType ->
-                        navigateToDownloadProgressDialog(attachment, attachmentIntentType, NewMessageFragment::class.java.name)
-                    },
-                    snackbarManager = snackbarManager,
-                )
+                if (it is Attachment) {
```

Comment:
Test if this works well : 
```suggestion
                if (it !is Attachment) return@AttachmentAdapter
```